playwright/manual-multiple.test.ts

```diff
@@ -0,0 +1,49 @@
+import { expect, test } from '@playwright/test';
+
+const viewport = { width: 1600, height: 1000 };
+
+const widths = ['360', '740', '980', '1300', '100%'];
+
+test.describe('Manual Multiple visual regression testing', () => {
+	for (const breakpoint of widths) {
+		test(`Manual Multiple: ${breakpoint}`, async ({ page }) => {
+			await page.setViewportSize(viewport);
+
+			await page.goto(
+				'https://guardian.github.io/commercial-templates/csr/manual-multiple/',
+				{
+					waitUntil: 'networkidle',
+				},
+			);
+
+			const referenceTemplateLocator = page
+				.frameLocator(`[name='width-${breakpoint}']`)
+				.locator('html');
```

Comment:
Selecting the html inside the iframe allows us to sidestep issues we can get as a result of padding sometimes being added to the bottom of the iframe, like here:
<img width="406" alt="Screenshot 2024-07-03 at 15 37 33" src="https://github.com/guardian/commercial-templates/assets/108270776/cf58efbd-cd5d-421e-b77a-04d67de98a3e">